backend/tests/users.test.ts

```diff
@@ -429,3 +429,23 @@ describe("Testing DELETE user", () => {
     expect(response.status).toBe(500);
   });
 });
+
+
+describe("Testing GET /users/pagination", () => {
+  test("Get 2 users after the second use", async () => {
+    const users = await request(app).get("/users");
+    const userid = users.body[1].id;
+    const response = await request(app).get("/users/pagination?limit=2&after=" + userid);
```

Comment:
Small syntactic sugar suggestion:
```suggestion
    const response = await request(app).get(`/users/pagination?limit=2&after=${userid}`);
```